blog.html

```diff
@@ -0,0 +1,277 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;600&display=swap"
+      rel="stylesheet"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Bakbak+One&family=Poppins:wght@400;500;600&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="all.css" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Blog</title>
+  </head>
+  <body>
+    <header class="header">
+      <a class="logo" href="index.html">Binabox</a>
+      <nav class="navigation">
+        <ul class="navigation-list">
+          <li class="navigation-item">
+            <a href="index.html" class="nav-link">Home</a>
+          </li>
+          <li>
+            <a href="blog.html" class="nav-link">Blog</a>
+          </li>
+          <li>
+            <a href="catalog.html" class="nav-link">Catalog</a>
+          </li>
+          <li>
+            <a href="contact-us.html" class="nav-link">Contact Us</a>
+          </li>
+        </ul>
+      </nav>
+      <div>
+        <input type="text" size="2" />
+        <a href="link" class="button-discord" target="_blank">Discord</a>
+        <button class="button-blue" type="button">Connect</button>
+      </div>
+    </header>
+    <main>
+      <section class="name-page">
+        <h1>Our blog</h1>
+        <ul class="blog-nav">
+          <li>
+            <a href="index.html" class="blog-nav-link">Home</a>
+          </li>
+          <li>
+            <a href="blog.html" class="blog-nav-link">Our blog</a>
+          </li>
+          <li>
+            <a href="basilisks.html" class="blog-nav-link">The Basilisks</a>
+          </li>
+        </ul>
+      </section>
+      <div class="blog-info">
+        <aside class="sidebar">
+          <div class="sidebar-box">
+            <p class="sidebar-title">Categories</p>
+            <ul class="sidebar-list">
+              <li><a href="#" class="sidebar-link">Our blog (12)</a></li>
+              <li><a href="#" class="sidebar-link">Metaverse (24)</a></li>
+              <li><a href="#" class="sidebar-link">Gaming (24)</a></li>
+              <li><a href="#" class="sidebar-link">Art (24)</a></li>
+              <li><a href="#" class="sidebar-link">Robot (24)</a></li>
+              <li><a href="#" class="sidebar-link">Lauchpad (24)</a></li>
+              <li><a href="#" class="sidebar-link">Crypto (430)</a></li>
+            </ul>
+          </div>
+          <div class="sidebar-box">
+            <p class="sidebar-title">Tag</p>
+            <ul class="sidebar-list">
+              <li><a href="#" class="sidebar-link">NFT</a></li>
```

Comment:
Please note that the 'Our blog (12)' and the'Tag' are different in design. Therefore, they should be given different classes.

Reply:
done